src/dnaio/_core.pyx

```diff
@@ -485,58 +452,57 @@ cdef class FastqIter:
     def __next__(self):
         cdef:
             object ret_val
-            Py_ssize_t name_start, name_end, name_length
-            Py_ssize_t sequence_start, sequence_end, sequence_length
-            Py_ssize_t second_header_start, second_header_end, second_header_length
-            Py_ssize_t qualities_start, qualities_end, qualities_length
-            char *name_end_ptr
-            char *sequence_end_ptr
-            char *second_header_end_ptr
-            char *qualities_end_ptr
+            char * name_start
```

Comment:
I know you have consistently used this style in your PRs whenever you changed or added any pointer declarations, but I think this is not an improvement. There [are arguments for both the `char* p` and `char *p` style](https://www.stroustrup.com/bs_faq2.html#whitespace), but using spaces on both sides of the `*` is the option that seems to be the most unconventional. CPython itself seems to use `char *p` most of the time.

Reply:
In my mind I read it as "char pointer name_start" hence the spaces. But I will conform to the `char *p` style. Sorry I forgot about this.

Reply:
Done